versions.mk

```diff
@@ -18,7 +18,7 @@ VERSION ?= v0.6.0
 vVERSION := v$(VERSION:v%=%)
 
 CUDA_VERSION := 12.3.2
-GOLANG_VERSION := 1.20.5
+GOLANG_VERSION := 1.22.1
```

Comment:
Just a note -- this go version bump resolves a number of high CVEs currently flagged in mig-manager ([scan results](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/cloud-native/containers/k8s-mig-manager/security?tag=v0.6.0-ubi8&architecture=arm64)).

Reply:
This was addressed in #53